model/task/task.go

```diff
@@ -1957,6 +1983,68 @@ func (t *Task) SetDisabledPriority(user string) error {
 	return t.DeactivateTask(user)
 }
 
+// DisableTasks is the same as (*Task).SetDisabledPriority but for many tasks.
+func DisableTasks(tasks []Task, caller string) error {
```

Comment:
Should we be consistent with naming `caller` the same as `user` in `SetDisabledPriority`?

Reply:
I changed `SetDisabledPriority` to use `caller`.

The DB models are a little inconsistent about user/caller because in some cases, the one performing the operation might be an actual Evergreen user while in others, it's initiated by an automated background process.